twilight-model/src/channel/message/component/select_menu.rs

```diff
@@ -1,27 +1,59 @@
-use crate::channel::message::ReactionType;
+use crate::channel::{message::ReactionType, ChannelType};
 use serde::{Deserialize, Serialize};
 
-/// Dropdown-style [`Component`] that renders belew messages.
+/// Dropdown-style [`Component`] that renders below messages.
 ///
 /// [`Component`]: super::Component
 #[derive(Clone, Debug, Eq, Hash, PartialEq)]
 pub struct SelectMenu {
+    /// An optional list of channel types.
+    ///
+    /// This option is only used for [channel select menus](SelectMenuType::Channel).
```

Comment:
Precedent language from [Channel](https://github.com/twilight-rs/twilight/blob/main/twilight-model/src/channel/mod.rs#L117) would be "This is only applicable to channel select menus", but this sounds okay too

Reply:
Changed :white_check_mark: